VideoLocker/src/main/java/org/edx/mobile/task/SetThreadReadTask.java

```diff
@@ -11,22 +11,15 @@
 
 public class SetThreadReadTask extends Task<DiscussionThread> {
     private final DiscussionThread thread;
-    private final boolean read;
 
     public SetThreadReadTask(@NonNull Context context,
-                             @NonNull DiscussionThread thread, boolean read) {
+                             @NonNull DiscussionThread thread) {
         super(context);
         this.thread = thread;
-        this.read = read;
     }
 
     @Override
     public DiscussionThread call() throws RetroHttpException {
-        return environment.getDiscussionAPI().setThreadRead(thread, read);
-    }
-
-    @Override
-    protected void onSuccess(DiscussionThread discussionThread) {
-        EventBus.getDefault().post(new DiscussionThreadUpdatedEvent(discussionThread));
```

Comment:
That's an implementation detail, which might change later. We should still be posting globally relevant events where they might make sense, regardless of whether the current implementation needs to handle it or not.